test/ids-action-sheet/ids-action-sheet-percy-test.ts

```diff
@@ -0,0 +1,19 @@
+import percySnapshot from '@percy/puppeteer';
+
+describe('Ids Action Sheet Percy Tests', () => {
+  const url = 'http://localhost:4444/ids-action-sheet/example.html';
+
+  it('should not have visual regressions when cancel button is not visible', async () => {
+    await page.setViewport({ width: 450, height: 800 });
+    await page.goto(url, { waitUntil: ['networkidle2', 'load'] });
+    await page.evaluate(() => {
+      (document.querySelector('#icon-button') as any).click();
+    });
+    await page.waitForSelector('ids-action-sheet[visible]');
+    await page.evaluate(() => {
+      const idsButton = document.querySelector('ids-action-sheet')?.shadowRoot?.querySelector('ids-button') as HTMLElement;
+      idsButton.style.display = 'none';
```

Comment:
Instead of hiding with CSS, I'd suggest using the same API you're using in the functional tests to hide the button, just to make the test a bit more accurate:

```suggestion
      idsButton.cancelBtnText = null;
```